itkwidgets/viewer.py

```diff
@@ -376,6 +382,23 @@ async def get_roi_image(self, scale: int = -1, name: str = 'Image') -> NgffImage
             )
         raise ValueError(f'No image data found for {name}.')
 
+    @fetch_value
+    async def get_roi_multiscale(self, name: str = 'Image') -> Multiscales:
+        """Build and return a new Multiscales NgffImage for the ROI.
+
+        :param name: Name of the loaded image data to use. 'Image', the
+        default, selects the first loaded image.
+        :type name:  str
+
+        :return: roi_multiscales
+        :rtype:  Multiscales NgffImage
+        """
+        if store := self.stores.get(name):
+            multiscales = from_ngff_zarr(store)
+            ngffImage = await self.get_roi_image(scale=0, name=name)
+            return to_multiscales(ngffImage, method=multiscales.method)
```

Comment:
Instead, can we iterate through `multiscales.images` and extract the roi image for each based on the roi region, as in the get_roi_image implementation, generate a new list, and create a new `Multiscales` with the roi `images` list, the same `metadata`, `scale_factors`, `method`, and `chunks`. This will prevent recomputation.